src/QMCHamiltonians/OrbitalImages.h

```diff
@@ -171,7 +171,7 @@ class OrbitalImages : public OperatorBase
   std::vector<std::vector<int>*> sposet_indices;
```

Comment:
I think we can use
`const std::shared_ptr<std::vector<std::vector<int>>>` to simplify codes.

Reply:
Is the ownership scope of the SPOSet really so murky?

Reply:
@ye-luo `sposet_indices` is set in `OrbitalImages::put(xmlNodePtr)`, so making it a const member may be difficult.

Reply:
If I understand correctly, const only applies to shared_ptr. You can still resize the vector.